datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/search/SearchForEntitiesByFormResolver.java

```diff
@@ -0,0 +1,89 @@
+package com.linkedin.datahub.graphql.resolvers.search;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.bindArgument;
+import static com.linkedin.datahub.graphql.resolvers.search.SearchUtils.getEntityNames;
+import static com.linkedin.datahub.graphql.resolvers.search.SearchUtils.mapInputFlags;
+
+import com.google.common.collect.ImmutableList;
+import com.linkedin.common.urn.UrnUtils;
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.generated.SearchForEntitiesByFormInput;
+import com.linkedin.datahub.graphql.generated.SearchResults;
+import com.linkedin.datahub.graphql.resolvers.ResolverUtils;
+import com.linkedin.datahub.graphql.resolvers.mutate.util.FormUtils;
+import com.linkedin.datahub.graphql.types.mappers.UrnSearchResultsMapper;
+import com.linkedin.entity.client.EntityClient;
+import com.linkedin.form.FormInfo;
+import com.linkedin.metadata.query.SearchFlags;
+import com.linkedin.metadata.query.filter.Filter;
+import com.linkedin.metadata.service.FormService;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.concurrent.CompletableFuture;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Search across entities by form information. Used in order to get entities assigned to specific
+ * forms as well as other details about the form and prompt states.
+ */
+@Slf4j
+@RequiredArgsConstructor
+public class SearchForEntitiesByFormResolver
```

Comment:
let's actually leave this resolver out as well - it's only used to serve the bulk filling out of form situation. I also am having second thoughts about whether we even want this as a new endpoint or just pass `formFilters` as an optional param to our classic `searchAcrossEntities` when we bring the bulk things to SaaS